README.md

```diff
@@ -34,89 +34,108 @@ Loon is a repository that is meant to be deployed as a standalone docker contain
 
 ## Use And Deployment
 
-All of the necessary applications are shipped in a single docker container. In the repository, you will see two docker directories: `docker` and `docker-local`. The main deployment will use the `docker` directory. The `docker-local` directory is a separate local version of Loon which we will discuss shortly.
+All of the necessary applications are shipped in a single docker container. We use a python script called "build.py" to initiate and simplify the process.
+
+**Note that docker and Python 3+ must be installed and running in order to use the "build.py" script**
+
+We start by making a configuration file called "config.json". Below is a template:
+
+```json
+{
+  "generalSettings": {
+    "useHttp": false,
+    "environment": "production",
+    "baseUrl": "localhost"
+  },
+  "mySqlSettings": {
+    "databaseName": "loon",
+    "databaseUser": "user",
+    "databasePassword": "user_pass",
+    "databaseRootPassword": "root_pass",
+    "sourceVolumeLocation": "/Users/bbollen23/loonar-data/mysql-data"
+  },
+  "minioSettings": {
+    "minioStorageAccessKey": "admin",
+    "minioStorageSecretKey": "minioadmin",
+    "sourceVolumeLocation": "/Users/bbollen23/loonar-data/minio-data"
+  },
+  "nginxSettings": {
+    "sourceVolumeLocation": "/Users/bbollen23/loonar-data/ssl",
+    "targetVolumeLocation": "/etc/nginx/ssl",
+    "certFileLocation": "certs/localhost.crt",
+    "keyFileLocation": "private/localhost.key"
+  }
```

Comment:
Are any of these optional? It seems like most of these are related to running https locally, which generally isn't needed.

If these are required, could you provide more info on generating these files.

When I tried building with the default config, I was getting build errors in `client.log`.

Reply:
Yes we can talk about this. Note that this will be changing hopefully so that the SSL stuff doesn't need to be volumes. Issue arises with some permissions, however, so I have to do some testing. This is only for HTTPS stuff -- if you set "useHttp" to true, you can delete this whole section from your config. To generate these, you can use OpenSSL or Certbot